src/components/header/List/List.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { List } from './List'
+
+export default {
+  title: 'List',
+  parameters: {
+    info: `
+USWDS 2.0 List component
+
+`,
+  },
+}
+
+const testItems = [
+  <a href="#linkOne" key="one">
+    <span>Test link</span>
+  </a>,
+  <a href="#linkTwo" key="two">
+    <span>Test Link Two</span>
+  </a>,
+]
+
+const keyPrefix = 'testItem'
+
+export const BasicNavList = (): React.ReactElement => (
+  <List
+    items={testItems}
+    keyPrefix={keyPrefix}
```

Comment:
I don't think it is strictly needed, but the key itself is required. I don't think it is the best id to only use the index for the key, which is why I added the key prefix: to help id the list element and what list it belongs to. 
https://medium.com/@adhithiravi/why-do-i-need-keys-in-react-lists-dbb522188bbb

Reply:
Ah, okay I see, thanks for explaining!  So FYI in terms the best practice of trying not to use an index as a key - by adding the prefix doesn't make the keys any more unique among siblings  than by using just the index (since the index is still used in `${keyPrefix} - i`). They both have the same potential issue (where indices could be repeated in the same list if items are reshuffling or adding/removing items quickly).